docs/app/pages/components/components-sections/tabs/card-tabs/card-tabs.component.html

```diff
@@ -0,0 +1,71 @@
+<ux-card-tabset [position]="position">
+
+    <ux-card-tab class="card" *ngFor="let tab of tabs">
+        <h3 class="card-title">{{ tab.title }}</h3>
+        
+        <div class="card-image" [ngClass]="tab.image"></div>
+        
+        <h3 class="card-value">{{ tab.value }}<small>{{ tab.unit }}</small></h3>
+        <h4 class="card-subtitle">{{ tab.subtitle }}</h4>
+
+        <ng-template uxCardTabContent>
+            <h4>{{ tab.title }}</h4>
+            <p>{{ tab.content }}</p>
+        </ng-template>
+    </ux-card-tab>
+
+</ux-card-tabset>
+
+<hr>
+
+<div class="row uxd-customize-example">
+    <div class="col-md-12">
+        <accordion>
+            <accordion-group class="accordion-chevron" heading="Customize Example...">
+                <div class="row uxd-customize-row">
+                    <div class="col-md-4 col-sm-12">
+                        <label>Position</label>
+                        <ux-radio-button [(value)]="position" option="top">top</ux-radio-button>
+                        <ux-radio-button [(value)]="position" option="bottom">bottom</ux-radio-button>
+                    </div>
+                </div>
+            </accordion-group>
+        </accordion>
+    </div>
+</div>
+
+<p>
+    Card tabs are defined inside a <code>ux-card-tabset</code> component using a <code>ux-card-tab</code> component.
+    Inside the <code>ux-card-tab</code> component insert any content you want to display in the card.
+</p>
+<p>
+    Also inside the <code>ux-card-tab</code> component an element with a <code>uxCardTabContent</code> directive should be defined which will contain
```

Comment:
Could we swap this around so that the directive defines the tab header and the remainder is the content? Then it would be consistent with the Bootstrap tabs:

````html
<tabset>
    <tab>
        <ng-template tabHeading>
            <span class="hpe-icon hpe-platform-chrome"></span> <span>Chrome</span>
        </ng-template>
        <p class="m-t-md">
            Google Chrome is a fast, secure and free web browser, built for the modern web.
        </p>
    </tab>
</tabset>
````